focuspoints.lrdevplugin/PentaxDelegates.lua

```diff
@@ -176,7 +180,7 @@ end
 
 function PentaxDelegates.fixCenter(points)
   for k,v in pairs(points) do
-    if v == 'Center (vertical)' or v == 'Center (horizontal)' then
+    if v == 'Center (vertical)' or v == 'Center (horizontal)' or v == 'Fixed Center' then
```

Comment:
I'm thinking there won't be many combinations for this, but I won't know for sure until I get my hands on more raw files. I think it's really strange how Pentax has at least 3 names for the exact same center point.